### DISCUSSION TITLE
Many things Not Working

### DISCUSSION THREAD

**Discussion by VictorM700** [elementary/installer#858] [Q&A]
Your installer does not have the option to install onto free partition, leaving my Fedora OS alone. I HAVE TO erase my whole drive. I tried to re-use the free partition, but the insrtaller fails. 

Your OS is the ONLY ONE that does not run under a virtual machine. What do think Windows users would do ? Erase the only OS they understand and install yours without trying it out under a vm first? Yesterfay I did install it under QEMU. And it freezes a lot, but you warned about it so I'll let it pass. But there is NO REASON why your OS cannot run under a vm.

Today I did the bare metal install . After installation, I went directly to ufw to setup the firewall. Then I start epiphany and it couldn't get onto the internet. And cannot do a dig, And yes I opened all the necessary ports.  

Your firewall app has no function. It can't create or delete firewall rules. So what is it for?  

Your repository does not have snapd. So I cannot use the snap version of firefox. So I went with your Store's flatpak version of firefox. But the flatpak apparmor profile is just a stub using unconfined. So it is useless. Ubuntu's snap firefox has apparmor and seccomp in addition to containerization. but you don't have snapd in your repository. Why? You are derived from Ubuntu. 

Absolute disgrace to open source.

**Comment by danirabbit** [comment]
OP this is not an acceptable way to participate in our community. Please review the code of conduct and remember to be respectful and kind. This is a volunteer driven project. Everyone is doing their best and cares deeply about doing a good job.

If you can change your mindset to being helpful and constructive, please feel free to reopen individual issue reports for each issue you've experienced as per our contributor guide: https://docs.elementary.io/contributor-guide/feedback/reporting-issues

In the meantime closing this discussion post since we can't track grouped reports with many unrelated topics like this and the overall tone here is not constructive
